packages/cli/src/routeGeneration/templates/express.hbs

```diff
@@ -66,7 +66,7 @@ export function RegisterRoutes(app: Router) {
             authenticateMiddleware({{json security}}),
             {{/if}}
             {{#if uploadFile}}
-            upload.single('{{uploadFileName}}'),
+            upload.fields({{json uploadFileName}}),
```

Comment:
This will not break single files?

Reply:
I also rewrite the uploadFileName field in routeGenerator.tsL105
Check [here](https://github.com/lukeautry/tsoa/pull/1541/files#diff-fa3f83789f7f648679cd056a5564dead51915ecc3903778f9808122e50be44eeR105-R108).

All exists tests are not been modified.